### ISSUE TITLE
plugin does not work with blog item

### ISSUE THREAD

**Issue by LaurentOngaro** [Perlkonig/grav-plugin-tablesorter#2]
Hi, 
I've tried to use this fantastic plugin on my website, without any code changes and at first it was not running. 
No error in Javascript console.

After a moment , I've discovered that the plugin only runs with a default page (which uses a default.md filename), and not with a blog item.
I think it could be changed easily, so I'l try to fix that by myself.
thanks for your work

**Comment by Perlkonig** [comment]
That should not be happening. I use it for my blog. Something else must be the problem.

* `blog.md`: https://www.perlkonig.com/blog/podcasts-2016
* `demo.md`: https://www.perlkonig.com/demos/tablesorter

**Comment by Perlkonig** [comment]
You're sure you're using `active` and `enabled` correctly?

**Comment by LaurentOngaro** [comment]
yes, I'm sure.
I've just put your demo code on a single page, and the table header code shown in the element inspector of my browser  is 
`<table id="tstableid1" class="tablesorter">`
 So the plugin is active and enabled
If I just switch the "page template" settings from "item" to "default", without changing anything else, the plugin works. With the inverse settings, it fails.
It's probably an issue with my template override for the blog item.
I'll check that tomorrow.

**Comment by Perlkonig** [comment]
Bonne chance! Let me know.

**Comment by LaurentOngaro** [comment]
I've found the cause.
In my template override, I've added a twig include used by the plugin "printfriendly". 
By removing this include, all runs well. 

So it's just an incompatibility issue between printfriendly and tablesorter.

In another hand, these seems rather logic because printfriendly is basically creating a copy of the page content into itself. I've already found incompatibilities between printfriendly and other plugins (anchors for instance).
I'll probably resolve that by adding a table filter in your plugin to exclude those created by printfriendly
